modules/web-console/frontend/app/components/ignite-chart/controller.js

```diff
@@ -81,6 +81,13 @@ export class IgniteChartController {
      * @param {{chartOptions: ng.IChangesObject<import('chart.js').ChartConfiguration>, chartTitle: ng.IChangesObject<string>, chartDataPoint: ng.IChangesObject<IgniteChartDataPoint>, chartHistory: ng.IChangesObject<Array<IgniteChartDataPoint>>}} changes
      */
     async $onChanges(changes) {
+        if (changes.resultDataStatus && ['WAITING', 'ERROR'].includes(changes.resultDataStatus.currentValue) && this.chart) {
+            this.chart.destroy();
```

Comment:
That's no problem, because ignite-chart depends on it's feeder to provide fault-tolerant streams. In case error occurs feeder's responsibility to handle it it send new data to the chart.